src/test/daml/Daml/Finance/Instrument/Bond/Test/Callable.daml

```diff
@@ -593,48 +593,48 @@ runFloatingFloorCapCallable6M = script do
     amount = 1.0
     electorIsOwner = False
 
-  -- Coupon date 1: Lifecycle and verify that there is an effect for one coupon.
+  -- Coupon date 1 (not a call date): Lifecycle and verify that there is an effect for one coupon.
   let
     expectedConsumed = []
     expectedProduced = [qty 0.0026888889 cashInstrumentCid]
-  (Some bondInstrumentAfterCoupon1, effectCid) <- electAndVerifyPaymentEffects (date 2022 Apr 15)
-    amount bondInstrument electorIsOwner issuer investor [publicParty] "NOT CALLED" [observableCid]
-    expectedConsumed expectedProduced
+  Some bondInstrumentAfterCoupon1 <- lifecycleAndVerifyPaymentEffects [publicParty]
+    (date 2022 Apr 15) bondInstrument issuer [observableCid] expectedConsumed expectedProduced
 
-  -- Coupon date 2: Lifecycle and verify that there is an effect for one coupon.
+  -- Coupon date 2 (call date): Lifecycle and verify that there is an effect for one coupon.
```

Comment:
Do we test the unhappy path? using e.g. `submitMustFail`

Reply:
I added a test for this as well.